authz/src/test/java/io/grpc/authz/FileWatcherAuthorizationServerInterceptorTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2022 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.authz;
+
+import static com.google.common.truth.Truth.assertThat;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class FileWatcherAuthorizationServerInterceptorTest {
+  @Test
+  public void invalidPolicyFailsAuthzInterceptorCreation() throws Exception {
+    File policyFile = File.createTempFile("temp", "json");
+    policyFile.deleteOnExit();
+    try (FileOutputStream outputStream = new FileOutputStream(policyFile)) {
```

Comment:
Feel free to use Guava's [`Files.write(byte[], File)`](https://guava.dev/releases/snapshot/api/docs/com/google/common/io/Files.html#write(byte%5B%5D,java.io.File)) and/or create a helper function to help make the test more terse.